clients/cobol-lsp-vscode-extension/src/services/copybook/downloader/DownloadStrategyResolver.ts

```diff
@@ -50,6 +63,16 @@ export class DownloadStrategyResolver {
     copybookName: CopybookName,
     documentUri: string,
   ): Promise<boolean> {
+    const e4e = await E4ECopybookService.getE4EClient(documentUri);
+    if (this.e4eDownloader && e4e) {
+      await this.e4eDownloader.downloadCopybookE4E(
+        copybookName,
+        documentUri,
```

Comment:
All the relevant information should be already in the `e4e`. If we get rid of the pgm name in the endevor elements path (which does not do anything useful at the moment) we should be able to drop the documentUri argument.